This is confusing these options are for the unreleased clang/llvm version 15.
no-opaque-pointers errors out 
Wno-unqualified-std-cast-call spams build log
diff -ruN v8-0.43.1.orig/build/config/compiler/BUILD.gn v8-0.43.1/build/config/compiler/BUILD.gn
--- v8-0.43.1.orig/build/config/compiler/BUILD.gn	1973-11-29 16:33:09.000000000 -0500
+++ v8-0.43.1/build/config/compiler/BUILD.gn	2022-06-18 21:45:54.792630868 -0400
@@ -1543,20 +1543,6 @@
         "-Wno-ignored-pragma-optimize",
       ]
 
-      if (!is_nacl) {
-        # TODO(https://crbug.com/1300731) Clean up and enable.
-        cflags += [ "-Wno-unqualified-std-cast-call" ]
-      }
-
-      if (!is_nacl && !(is_chromeos ||
-                        default_toolchain == "//build/toolchain/cros:target")) {
-        # TODO(https://crbug.com/1316298): Re-enable once test failure is figured out
-        cflags += [
-          "-Xclang",
-          "-no-opaque-pointers",
-        ]
-      }
-
       if (is_fuchsia) {
         # TODO(https://bugs.chromium.org/p/fuchsia/issues/detail?id=77383)
         cflags += [ "-Wno-deprecated-copy" ]
